conduct.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Code of conduct
+layout: default
+---
+
+# Code of conduct
+
+---
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and
+orientation.
+
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is implicitly or explicitly representing the project or community.
+
+## Standards of behaviour
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
```

Comment:
I'd say no. If someone is intentionally misgendering someone abusively, it's a problem and we'll sort it out, but forcing that on everyone isn't necessary imo.